.github/workflows/msbuild.yml

```diff
@@ -51,8 +49,14 @@ jobs:
     - name: Check whitespace and style
       working-directory: dafny
       run: dotnet tool run dotnet-format -w -s error --check Source/Dafny.sln --exclude Dafny/Scanner.cs --exclude Dafny/Parser.cs
-      
+  
+  check-deep-tests:
+    uses: ./.github/workflows/check-deep-tests-reusable.yml
+    if: "!(contains(github.event.pull_request.labels.*.name, 'skip-deep-tests-check'))"
```

Comment:
Nice addition! Documentation changes, for example, would be nice to not put through the whole battery of tests.

Reply:
Ah, but notice the feature here is to skip checking that the deep tests have run successfully on the nightly build, to deal with the edge case where a previous PR broke only the deep tests and you need another PR to fix them.

I think you're thinking of a label like `skip-integration-tests` that wouldn't run the integration tests at all on a PR. That's a reasonable feature request too and we could follow up with that. I was a little nervous about it because there are some Dafny files hiding in the `docs` folder tree with lit tests that include them.